content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -692,13 +692,8 @@ spec:
 
 ## Volume Cloning
 
-{{< feature-state for_k8s_version="v1.16" state="beta" >}}
```

Comment:
Should we keep the feature-state, but change it to GA/stable?

Reply:
I'm fine with either approach. It's part of a long page; on the other hand, it's handy for readers to know that the feature is new.

Reply:
I had been following the pattern of other features that GA and just dropping it.  It seems like having it for initial GA is fine, but also it doesn't seem like past features had that.  

Let me know if you prefer it be added, otherwise I'll leave it out.